src/routes/routes.tsx

```
@@ -97,22 +101,22 @@
-      <ContactUs />
-    ),
+    element: <ContactUs />,
     errorElement: <ErrorPage />,
   },
 
   {
     path: '/auth',
-    element: (<AuthProvider><PageWrapper children={<CommonLayout />} /></AuthProvider>),
+    element: (
+      <AuthProvider>
+        <PageWrapper children={<CommonLayout />} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)